tests/Unit/UserTest.php

```diff
@@ -45,7 +45,7 @@ public function testCreateUser(){
     $users = json_decode($response->content(),true)["data"]["users"];
 
     //create a new user 
-    $response = $this->callAuthenticated('GET', '/users/create',["name"=>"Brian Ludwig, JR","password"=>"Harper86","email"=>"faker@poop.com"]);
+    $response = $this->callAuthenticated('GET', '/users/create',["name"=>"Brian Ludwig, JR","password"=>"Harper86","email"=>"faker@poop.com", "permission" => "standard"]);
```

Comment:
That is outside the scope of this feature. That will be handled when I have guards and policies implemented.